### PR TITLE
FormTokenField: Deprecate 36px default size

### PR DESCRIPTION
**Pull request by hbhalodia**
Part of #65751

## What?

Deprecate the 36px default size on FormTokenField.

## Testing Instructions

- Unit tests pass
- Storybook stories should not log console warnings
- All code snippets in documentation (JSDoc, Storybook, README) should have the `__next40pxDefaultSize` prop enabled

### CODE REVIEW

File: packages/components/src/form-token-field/stories/index.story.tsx [hunk 1]
```diff
@@ -64,6 +64,7 @@ Default.args = {
 	label: 'Type a continent',
 	suggestions: continents,
 	__nextHasNoMarginBottom: true,
+	__next40pxDefaultSize: true,
```
Comment (mirka) [inline packages/components/src/form-token-field/stories/index.story.tsx:67]: Prop is missing in the Async story.

**Review by mirka — COMMENTED** [review]
There was one spot missing, but otherwise looking good!

File: packages/components/src/form-token-field/stories/index.story.tsx (hunk 1 above)
Comment (hbhalodia) [reply packages/components/src/form-token-field/stories/index.story.tsx:67]: Hi @mirka, This is now added.

**Review by mirka — APPROVED** [review]
Thank you! 🚀

**Pull request merged**